content/ai/concepts/computer-vision/computer_vision.md

```diff
@@ -0,0 +1,86 @@
+---
+Title: 'AI for Computer Vision'
+Description: 'Computer vision is a subfield of artificial intelligence, which specialises in recognition, extraction and processing of image data to simulate human vision. '
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Artificial Intelligence'
+Tags:
+  - 'AI'
+  - 'Image Classification Model'
+  - 'Image Generation Model'
+  - 'Prompt Engineering'
+  - 'Image Labelling'
+  - 'Deep Learning'
+  - 'Classification'
+  - 'Facial Recognition'
+  - 'Machine Learning'
```

Comment:
Make sure you use the tags present [here](https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If you want then add these tags in the `tags.md` file